samples/animegan/animegan_converter.py

```diff
@@ -0,0 +1,27 @@
+from typing import Any, List, Optional, Tuple
+
+import cv2
+import numpy as np
+
+from savant.base.converter import BaseAttributeModelOutputConverter
+from savant.base.model import AttributeModel
+
+
+class AnimeganConverter(BaseAttributeModelOutputConverter):
```

Comment:
We can add to savant/converter 
```
class ModelRawOutputConverter(BaseAttributeModelOutputConverter):
    def __call__(...) -> List[Tuple[str, Any, Optional[float]]]:
        return [
            (model.output.attributes[i].name, output, 1.0)
            for i, output in enumerate(output_layers)
        ]
```
And use it for such tasks (GAN, SuperResolution), moving needed model output scaling to overlay pyfunc.
@bwsw ?

Reply:
There is such converter in https://github.com/insight-platform/Savant/pull/536/commits/29449b452c0d30114f48c0e4a9ab86e0341df01b

Reply:
I'll leave it as is for this demo.